extension/chrome/skin/inactive-state.css

```diff
@@ -205,17 +205,24 @@ notification .messageImage:-moz-window-inactive {
  * id: treestyletab@piro.sakura.ne.jp
  * URL: https://addons.mozilla.org/firefox/addon/tree-style-tab/ */
 
+#main-window:-moz-window-inactive #TabsToolbar.treestyletab-tabbar-toolbar[orient="vertical"] .tabbrowser-tab[selected] {
```

Comment:
I will switch from `[selected]` to `[visuallyselected]` once I tested everything, didn't know that was a thing.